### ISSUE TITLE
React-native wrappedValidateZSchemaStorage missing informations

### ISSUE THREAD

**Issue by arekm213** [pubkey/rxdb#4898]
```
    "rxdb": "14.15.1",
    "rxdb-premium": "14.15.1",
    "react-native": "0.68.7",
```
Hello, we are using rxdb in react-native project with hermes engine and we are not able to use ajv schema validation, so we decided to switch to using **validate-z-schema** using built in **wrappedValidateZSchemaStorage**, which works but it seems like **title** and **description** are missing in **validationErrors** in logs

Logs we got from testing it:
```
[database] [state_syncing_key_value_pairs] upsert threw error {"error": [RxError (VD2): RxError (VD2):
object does not match schema
Given parameters: {
collection:"state_syncing_key_value_pairs"
writeError:{
  "status": 422,
  "isError": true,
  "documentId": "xd",
  "writeRow": {
    "document": {
      "id": "xd",
      "value": 123,
      "_meta": {
        "lwt": 1692031369974.01
      },
      "_deleted": false,
      "_attachments": {},
      "_rev": "1-rxlrzqgsan"
    },
    "previous": null
  },
  "validationErrors": [
    {
      "title": null,
      "description": null,
      "message": "Expected type string but found type integer"
    }
  ]
}}]}
 WARN  Possible Unhandled Promise Rejection (id: 0):
RxError (VD2): RxError (VD2):
object does not match schema
Given parameters: {
collection:"state_syncing_key_value_pairs"
writeError:{
  "status": 422,
  "isError": true,
  "documentId": "xd",
  "writeRow": {
    "document": {
      "id": "xd",
      "value": 123,
      "_meta": {
        "lwt": 1692031369974.01
      },
      "_deleted": false,
      "_attachments": {},
      "_rev": "1-rxlrzqgsan"
    },
    "previous": null
  },
  "validationErrors": [
    {
      "title": null,
      "description": null,
      "message": "Expected type string but found type integer"
    }
  ]
}}
RxError (VD2):
object does not match schema
Given parameters: {
collection:"state_syncing_key_value_pairs"
writeError:{
  "status": 422,
  "isError": true,
  "documentId": "xd",
  "writeRow": {
    "document": {
      "id": "xd",
      "value": 123,
      "_meta": {
        "lwt": 1692031369974.01
      },
      "_deleted": false,
      "_attachments": {},
      "_rev": "1-rxlrzqgsan"
    },
    "previous": null
  },
  "validationErrors": [
    {
      "title": null,
      "description": null,
      "message": "Expected type string but found type integer"
    }
  ]
}}
```

Could you help us to make this logs more readable so the schema validation is usable in react-native too?
Thank you!

**Comment by pubkey** [comment]
Not all validation plugins return the same type of error.
If validate-z-schema does not add a description, you need to make a PR at this library, not RxDB.

**Comment by arekm213** [comment]
I tried logging what error data z-schema validation returns, and it also returns `path` which is quite useful, and its not exposed right now, do you think its ok to add it to the plugin?

**Comment by pubkey** [comment]
Closing this, fixed in #4900
